opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -239,7 +239,7 @@ class SimulatorFullyImplicitBlackoilEbos
 
             // write the inital state at the report stage
             if (timer.initialStep()) {
-                output_writer_.writeTimeStep( timer, state, well_state, solver->model() );
+                //output_writer_.writeTimeStep( timer, state, well_state, solver->model() );
```

Comment:
yeah, that what what I would have preferred, too. the problem is that this would have prevented the current master branch of opm-output. Anyway: not a big deal.